db/migrate/20150312082018_create_videos.rb

```diff
@@ -0,0 +1,13 @@
+class CreateVideos < ActiveRecord::Migration
+  def change
+    create_table :videos do |t|
+      t.string :video_id
+      t.string :amara_link
+      t.string :title
+      t.integer :priority, :default => 0
+      t.string :description, :default => 'No description has been written.'
```

Comment:
This should not be the default description. What we want instead is for the method to check if this is empty.